packages/circuit-ui/components/NotificationBanner/NotificationBanner.module.css

```diff
@@ -24,13 +33,30 @@
 .content {
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
   max-width: 640px;
-  max-width: max(640px, 66vw);
```

Comment:
`max-width: max(640px, 66vw);`
This condition made the text stretche over a very long width with viewport is large (desktop and above).